combiner/lists/MML_SVG-full.lis

```diff
@@ -0,0 +1,14 @@
+jax/input/MathML/config.js
+jax/output/SVG/config.js
+jax/output/FastHTML/config.js
+extensions/mml2jax.js
+extensions/MathEvents.js
+extensions/MathZoom.js
+extensions/MathMenu.js
+jax/element/mml/jax.js
+extensions/toMathML.js
```

Comment:
Again, the `jax/input/MathML/jax.js` should be included here, and probably in `MML_SVG` below.